eradiate/radprops/rad_profile.py

```diff
@@ -522,26 +534,30 @@ def eval_sigma_a(self, spectral_ctx):
         """
         Evaluate absorption coefficient given spectral context.
         """
-        wavelength = spectral_ctx.wavelength
-        profile = self.eval_thermoprops_profile()
+        if self.absorption_switch is True:
```

Comment:
`is True` is overspecified. The following is valid and encouraged:

```suggestion
        if self.has_absorption:
```